docs/syncing-spicedb-repos.md

```diff
@@ -0,0 +1,80 @@
+# Syncing SpiceDB Repositories
+
+We manage two forks from authzed: one for the [SpiceDB](https://github.com/project-kessel/spicedb) project itself and one for the [SpiceDB Operator](https://github.com/project-kessel/spicedb-operator). This document outlines the steps needed to sync our forks with their upstream counterparts.
+
+## Steps to Sync Forked Repositories
+
+Follow these steps to sync your forked repository with the upstream repository:
+
+1. **Add the Upstream Remote**  
+    Ensure the upstream repository is added as a remote. Run the following command to add it:  
+    ```bash
+    git remote add upstream <upstream-repo-url>
+    ```  
+
+2. **Fetch Upstream Changes**  
+    Fetch the latest changes from the upstream repository:  
+    ```bash
+    git fetch upstream
+    ```
+
+3. **Create a New Branch from the Desired Release Tag**  
+    Identify the release tag you want to sync from the upstream repository. Create a new branch based on this tag:  
+    ```bash
+    git checkout -b sync-upstream-<tag> tags/$TAG
+    ```  
+    Replace `<tag>` with the desired release tag.
+
+4. **Switch to a New Branch for Rebasing**  
+    Create and switch to a new branch from our main where you will apply the rebase:  
+    ```bash
+    git checkout -b rebase-upstream-<tag> main
+    ```
+
+5. **Rebase onto the Sync Branch**  
+    Rebase your new branch onto the branch created from the upstream release tag:  
+    ```bash
+    git rebase sync-upstream-<tag>
+    ```
+
+6. **Resolve Any Conflicts**  
+    If there are conflicts during the rebase, resolve them manually. After resolving conflicts, continue the rebase process:  
+    ```bash
+    git rebase --continue
+    ```
+
+    You may run into issues with deleted files that we have removed in our fork. If you encounter this, you can skip the commit with the following command:  
+    ```bash
+    git rebase --skip
```

Comment:
It seems like skipping the commit is not the right thing. I would have thought we could do like in https://stackoverflow.com/questions/79152436/how-to-git-pull-rebase-when-i-have-deleted-files-locally:
```
git rm ...
git rebase --continue
```

Although I may be misunderstanding the context here.